### ISSUE TITLE
[Upload] When a user try to upload an attachment, it fails

### ISSUE THREAD

**Issue by mdartic** [locokit/locokit#183]
This is related to how we sent data from the front, or how we retrieve it from the back.

Check the diff between `workspace_id` and `workspaceId` :-)

**Comment by mdartic** [comment]
My fix is not working. Not so easy.

I think there is an issue during the computation of abilities for attachment + users `USER` / `CREATOR`.

The error we have is a `406` : `Your request cannot be handled. Your attachment seems not be recognized.`.
It's related to the following code.

```ts
      /**
       * If there is an attachment id,
       * we're on a update/patch/remove request.
       *
       * We'll retrieve workspace_id by trying to retrieve the current item.
       */
      let currentWorkspaceId = workspaceId
      if (attachmentId) {
        const currentAttachment = await services.attachment._get(attachmentId) as LckAttachment
        currentWorkspaceId = currentAttachment.workspace_id
      }

      if (!currentWorkspaceId) {
        throw new NotAcceptable('Your request cannot be handled. Your attachment seems not be recognized.')
      }
```

My idea was the front end was not sending correctly the `workspace_id` query param.
But with the [latest release](https://github.com/locokit/locokit/releases/tag/v0.7.5-pr.184) the error is not the same, and it's related to the fact we DON'T send the workspace id anymore...

@Alketta please I let you go through this issue for @bruhnild

**Comment by Alketta** [comment]
This fix  #186 causes a new bug, it is impossible to delete a file with superadmin role.
